.github/workflows/integration-tests.yml

```diff
@@ -79,6 +79,9 @@ jobs:
       - name: Node Plugin
         run: go test -v ./node/plugin/tests
 
+      - name: eigenda-client
```

Comment:
How long does it take to run? If too slow, we may only run it on master branch i.e. when PR is merged.

Reply:
@samlaf

Reply:
hmm depends on batcher basically. batching interval is 10 mins on holesky testnet right now?
I think we should remove waiting for finalization.. that's just dumb to test I feel, and adds a 12 minute interval. And its testing the disperser itself more so than the client.

Reply:
set waitForFinalization=false: [eca995e](https://github.com/Layr-Labs/eigenda/pull/820/commits/eca995e4700fb173a95deceec46ce14ed084e55e)
Ran locally and it took <2mins. Let's see here. Think it depends on when in batching interval we send the request, but should hopefully be < 10 mins otherwise we'll need to increase default 10min golang test limit just for this test.